src/main/java/org/kuali/kfs/gl/batch/service/impl/CuScrubberProcessImpl.java

```diff
@@ -309,4 +313,26 @@ protected TransactionError setupEntryWithPotentialContinuation(OriginEntryFull c
         return null;
     }
 
+    /**
+     * Overridden to also restrict plant indebtedness based on document type,
+     * by means of a custom PLANT_INDEBTEDNESS_DOCUMENT_TYPES parameter.
+     * 
+     * @see org.kuali.kfs.gl.batch.service.impl.ScrubberProcessImpl#processPlantIndebtedness(
+     * org.kuali.kfs.gl.businessobject.OriginEntryInformation, org.kuali.kfs.gl.service.ScrubberReportData)
+     */
+    @Override
+    protected String processPlantIndebtedness(OriginEntryInformation scrubbedEntry, ScrubberReportData scrubberReport) {
+        // Make sure the entry was from a document that supports plant indebtedness, similar to the logic from the processCapitalization() method.
+        ParameterEvaluator documentTypeCodes = (parameterService.getParameterValueAsBoolean(ScrubberStep.class,

```

Comment:
Code looks good but I was wondering if readability could be improved a bit by extracting the plant indebtedness indicator into a boolean variable. The documentTypeCodes variable name is also a bit misleading since it is an evaluator rather than a collection of codes.

The if condition for returning null is also a bit unclear. null is returned if plant indebtedness indicator is false, scrubbed value is null or scrubbed entry doc type is not in the PLANT_INDEBTEDNESS_DOC_TYPE_CODES param. Checking if documentTypeCodes is null is hiding a bit the logic behind the decision and at first read without checking how the value is built makes you think that maybe the PLANT_INDEBTEDNESS_DOC_TYPE_CODES param is null/empty/does not exist and hence the docTypesCodes collection is null.